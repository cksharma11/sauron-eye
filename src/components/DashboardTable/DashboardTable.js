import React from "react";
import DashboardHeader from "../DashboardHeader/DashboardHeader";
import Pushers from "../Pushers/Pushers";

const DashboardTable = ({ dashboard }) => {
  return (
    <table border="1">
      <thead>
        <DashboardHeader
          headings={["Pusher", "SHA", "Passed/Total", "Graph"]}
        />
        <Pushers dashboard={dashboard} />
      </thead>
    </table>
  );
};

export default DashboardTable;
