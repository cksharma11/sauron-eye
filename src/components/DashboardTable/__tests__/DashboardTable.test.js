import React from "react";
import { shallow, configure } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import DashboardTable from "../DashboardTable";

configure({ adapter: new Adapter() });

describe("DashboardTable", () => {
  it("should render properly", () => {
    const wrapper = shallow(<DashboardTable dashboard={[{}, {}]} />);
    expect(wrapper.debug()).toMatchSnapshot();
  });
});
