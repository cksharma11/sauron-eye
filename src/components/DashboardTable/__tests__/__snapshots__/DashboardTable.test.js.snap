// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DashboardTable should render properly 1`] = `
"<table border=\\"1\\">
  <thead>
    <DashboardHeader headings={{...}} />
    <Pushers dashboard={{...}} />
  </thead>
</table>"
`;
